src/main/java/uk/ac/ebi/ena/readtools/webin/cli/rawreads/FastqScanner.java

```diff
@@ -128,7 +129,8 @@ private QualityNormalizer getQualityNormalizer(RawReadsFile rf ) {
         {
             String stream_name = rf.getFilename();
 
-            AutoNormalizerDataSpotProducer dp = new AutoNormalizerDataSpotProducer(is, "", rf.getFilename());
+            AutoNormalizerDataSpotProducer dp = new AutoNormalizerDataSpotProducer(
+                    is, runDuration == null ? null : runDuration, "", rf.getFilename());

```

Comment:
runDuration == null ? null : runDuration

could be simplified to:

runDuration